### ISSUE TITLE
Display step count from Pebble health

### ISSUE THREAD

**Issue by SimonKagstrom** [cfg1/pebble-mss#1]
Thanks for creating this watchface - easily my favorite!

Anyway, it would be nice if the new Pebble health step API could be integrated into mss, to display the step count on the watchface.

**Comment by cfg1** [comment]
Nice idea, but there is no space at the moment ...
Am 04.02.2016 12:44 schrieb "Simon Kagstrom" notifications@github.com:

**Comment by maxim-zapryanov** [comment]
Can't you make an option to replace for example  humidity and pressure with info from pebble health? Or some other of the info?
Otherwise awesome watchface! Love it!

**Comment by cfg1** [comment]
Yes I need to do such a thing. But I have not much time at the moment 
for programming on my watchface.

Am 06.02.2016 um 08:34 schrieb MaxZZZed:
